addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2090,6 +2091,19 @@ var SnippetsMenu = Widget.extend({
             }],
         }).open();
     },
+    /**
+     * @private
+     */
+    _onMouseDown: function () {
+        $('#wrapwrap').addClass('disable_pointer_events');
```

Comment:
- Never search for the DOM for random elements like that. Use the `getEditableArea` function like the rest of the code
- Classes of Odoo should follow the convention `o_` + app identifier + the rest. The identifier for web_editor is we, so here `o_we_no_pointer_events`